src/org/centos/pipeline/PackagePipelineUtils.groovy

```diff
@@ -306,21 +321,37 @@ def checkBranch() {
     return result
 }
 
-def packageMetrics() {
-    def tags = [:]
-    def fields = [:]
-    def measurement = env.fed_repo
-    tags['build_result'] = currentBuild.result
-    fields['build_time'] = currentBuild.getDuration().toString()
-
-    return [measurement, tags, fields]
+/**
+ * Parse the repo name from the request list.
+ * Sets env.fed_repo
+ * @param request
+ * @return
+ */
+def repoFromRequest(def request) {
+    if (!env.fed_repo) {
+        try {
+            def pkgUrlTok = request[0].tokenize('/')
```

Comment:
I am looking a little more at the message example you send me. I think we can just use name. It seems name, version, and release are all in the message, and name is the repo name *I think*. There is some discrepancy like for krb5 that I am not entirely clear on, so I think we should check with Miroslav on if we can use rpm name as the repo or we need to do this